install/management/appliance.go

```diff
@@ -301,18 +279,22 @@ func (d *Dispatcher) createAppliance(conf *configuration.Configuration) error {
 			Value: fmt.Sprintf("-serveraddr=0.0.0.0 -port=%s -port-layer-port=8080 %s", d.DockerPort, d.dockertlsargs)})
```

Comment:
Eventually, they should be part of executor config, but so far I'm not clear for how exectutor config will be used, so do not change that part.
Do you that config this time?